readme.md

```diff
@@ -1,34 +1,57 @@
-# Azure Industrial IoT
+# Azure Industrial IoT Cloud Platform
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT) [![Build Status](https://msazure.visualstudio.com/One/_apis/build/status/Custom/Azure_IOT/Industrial/Components/Azure.Industrial-IoT?branchName=master)](https://msazure.visualstudio.com/One/_build/latest?definitionId=86580&branchName=master)
 
-## Features
+## Welcome!
 
-### Discover, register and manage your Industrial Assets with Azure
+The Azure Industrial IoT Cloud Platform is a Microsoft product that has fully embraced openness. We use Azure's managed Platform as a Service (PaaS) services, open-source software leveraging the MIT license throughout, open international standards for communication (OPC UA, MQTT) and interfaces (Open API) and open industrial data models (OPC UA) on the edge and in the cloud. This means you are **not** locked in to our platform when using it!
 
```

Comment:
The comment of "Not locked into our platform" is so negative - can we positively rephrase, e.g. The platform does not depend on proprietary extensions or so.